src/game/engine/unit.cpp

```diff
@@ -345,24 +379,29 @@ void UnitClass::Enter_Idle_Mode(BOOL a1)
 #endif
 }
 
+/**
+ *
+ *
+ */
 BOOL UnitClass::Start_Driver(coord_t &dest)
 {
-#ifdef GAME_DLL
-    DEFINE_CALL(func, 0x0057F398, BOOL, UnitClass *, coord_t &);
-    return func(this, dest);
-#else
+    if (DriveClass::Start_Driver(dest) && m_IsActive) {
+        Mark_Track(dest, MARK_PUT);
+        return true;
+    }
     return false;
-#endif
 }
 
+/**
+ *
+ *
+ */
 BOOL UnitClass::Offload_Ore_Bail()
 {
-#ifdef GAME_DLL
-    DEFINE_CALL(func, 0x005808D8, BOOL, UnitClass *);
-    return func(this);
-#else
+    if (m_BailCount) {
```

Comment:
Coding convention says this should be `m_BailCount != 0`, be explicit with the check for the type of value.